[quant][graphmode][fx] Add support for cat

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43377 [quant][graphmode][fx] Add support for instance_norm
* #43376 [quant][graphmode][fx] Add support for layer_norm
* #43375 [quant][graphmode][fx] Add support for elu
* #43374 [quant][graphmode][fx] Add support for hardswish
* #43373 [quant][graphmode][fx] Add support for mul and mul relu
* #43372 [quant][grapphmode][fx][test][refactor] Refactor quantized add test
* #43335 [quant][graphmode][fx] Add support for batchnorm relu
* #43334 [quant][graphmode][fx] Add support for batchnorm
* **#43333 [quant][graphmode][fx] Add support for cat**
* #43332 [quant][graphmode][fx] Add support for add relu
* #43331 [quant][graphmode][fx] Add support for add
* #43287 [quant][graphmode][fx] Add support for conv module + relu

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243562](https://our.internmc.facebook.com/intern/diff/D23243562)

jerryzh168 committed:
[quant][graphmode][fx] Add support for cat

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for cat"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for cat"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243562](https://our.internmc.facebook.com/intern/diff/D23243562)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for cat"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243562](https://our.internmc.facebook.com/intern/diff/D23243562)

[ghstack-poisoned]

z-a-f commented:
nit: Going forward, can we keep the tests in the same PRs where associated code is. That way it is easier to bisect

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for cat"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243562](https://our.internmc.facebook.com/intern/diff/D23243562)

[ghstack-poisoned]
